mlir/utils/performance/bert-configs

```diff
@@ -0,0 +1,18 @@
+# This file has been generated with the following command:
+# convertRocBlasToPerfRunner.py -c bert-configs-raw --output-file bert-configs
+
+-transA false -transB false -g 64 -m 1024 -n 384 -k 1024 -t f32
```

Comment:
Hold on, rocBLAS is column-major but these configs are row-major, so these need to be backwards